service/docs/source/GEOPM_CXX_MAN_PlatformIO.3.rst

```diff
@@ -36,180 +36,121 @@ Link with ``-lgeopmd``
 
 .. code-block:: c++
 
-    struct geopm_request_s {
-        int domain;
-        int domain_idx;
-        char name[NAME_MAX];
-    };
```

Comment:
This should be in the man page since it is an input when calling PlatformIO::batch_server()